postgresql.sql

```diff
@@ -0,0 +1,15 @@
+--http://sqlfiddle.com/#!15/9eecb7db59d16c80417c72d1e1f4fbf1/11608
+--Runs with postgresql 9.3
+--This will not match with the expected output because of the column header called "FizzBuzzOrBoth" !
```

Comment:
Is there a way to make it match the expected output? (I'm fairly certain there is.)